api/tests/acceptance/scripts/bigint/answer/copy-newly-inserted-rows-to-answerId-tmp-bigint-tables_test.js

```diff
@@ -0,0 +1,86 @@
+const {
+  addConstraintsAndIndexesToAnswersKeBigintTemporaryTables,
+} = require('../../../../../scripts/bigint/answer/add-constraints-and-indexes-to-answers-ke-temporary-tables-with-bigint');
+const {
+  copyNewlyInsertedRowsToAnswerIdTmpBigintTables,
+} = require('../../../../../scripts/bigint/answer/copy-newly-inserted-rows-to-answerId-tmp-bigint-tables');
+
+const { expect, knex } = require('../../../../test-helper');
+
+const DatabaseBuilder = require('../../../../../db/database-builder/database-builder');
+const databaseBuilder = new DatabaseBuilder({ knex });
+
+describe('#copyNewlyInsertedRowsToAnswerIdTmpBigintTables', function () {
+  it('should add triggers into answers and KE bigint tables to copy newly inserted rows', async function () {
+    // given
+    await knex.raw('DROP TABLE IF EXISTS "bigint-migration-settings"');
+    await knex.raw('DROP TABLE IF EXISTS "knowledge-elements_bigint"');
+    await knex.raw('DROP TABLE IF EXISTS "answers_bigint"');
+    await knex.raw('DROP TRIGGER IF EXISTS trg_answers ON answers');
+    await knex.raw('DROP TRIGGER IF EXISTS "trg_knowledge-elements" ON "knowledge-elements"');
+
+    await createAnswersBigintMigrationDatabaseStructures(knex);
+
+    // when
+    const { id: assessmentId } = databaseBuilder.factory.buildAssessment();
+    const { id: firstAnswerId } = databaseBuilder.factory.buildAnswer({ assessmentId });
+    databaseBuilder.factory.buildKnowledgeElement({ assessmentId, answerId: firstAnswerId });
+    const { id: secondAnswerId } = databaseBuilder.factory.buildAnswer({ assessmentId });
+    databaseBuilder.factory.buildKnowledgeElement({ assessmentId, answerId: secondAnswerId });
+
+    await databaseBuilder.commit();
+
+    await copyAnswerKeDataToTemporaryTables();
+    await addConstraintsAndIndexesToAnswersKeBigintTemporaryTables();
+    await copyNewlyInsertedRowsToAnswerIdTmpBigintTables();
+
+    // then
+    const { rows: triggersByTable } = await knex.raw(
+      `SELECT  event_object_table AS table_name ,trigger_name
+             FROM information_schema.triggers
+             GROUP BY table_name , trigger_name
+             ORDER BY table_name ,trigger_name
+            `
+    );
+    expect(triggersByTable).to.deep.equal([
+      {
+        table_name: 'answers',
+        trigger_name: 'trg_answers',
+      },
+      {
+        table_name: 'knowledge-elements',
+        trigger_name: 'trg_knowledge-elements',
+      },
+    ]);
+    const { rows: answersBigintCount } = await knex.raw(`SELECT COUNT(1) FROM "answers_bigint"`);
+    expect(answersBigintCount[0].count).to.equal(2);
+    const { rows: knowledgeElementsBigintCount } = await knex.raw(`SELECT COUNT(1) FROM "knowledge-elements_bigint"`);
+    expect(knowledgeElementsBigintCount[0].count).to.equal(2);
+
+    const { answerBigIntMaxId } = await knex('answers_bigint').max('id as answerBigIntMaxId').first();
```

Comment:
Proposition: 
- isoler cette assertion dans un `it('should persist last base table identifier into settings table` 
- mettre le when dans un `beforeEach` si besoin de perf
```suggestion
    const { answerBigIntMaxId } = await knex('answers_bigint').max('id as answerBigIntMaxId').first();
    const { startsAtId: currentStartsAtId } = await knex('bigint-migration-settings')
      .where({
        tableName: 'answers',
      })
      .select('startsAtId')
      .first();
    const expectedStartsAtId = answerBigIntMaxId + 1;
    expect(currentStartsAtId).to.equal(expectedStartsAtId);
```